curvefs/src/mds/topology/topology_token_generator.h

```diff
@@ -38,12 +38,9 @@ class TopologyTokenGenerator {
     virtual std::string GenToken() = 0;
 };
 
-
 class DefaultTokenGenerator : public TopologyTokenGenerator {
  public:
-    DefaultTokenGenerator() {
-        std::srand(std::time(nullptr));
-    }
+    DefaultTokenGenerator() { std::srand(std::time(nullptr)); }
```

Comment:
std::srand should only call once

Reply:
`void MDS::InitTopology(const TopologyOption& option) {
    auto topologyIdGenerator = std::make_shared<DefaultIdGenerator>();
    auto topologyTokenGenerator = std::make_shared<DefaultTokenGenerator>();`

only call once here